cs.py

```diff
@@ -115,6 +117,8 @@ def _request(self, command, json=True, opcode_name='command',
             kwargs['response'] = 'json'
         if 'page' in kwargs or fetch_list:
             kwargs.setdefault('pagesize', 500)
+        if self.cert:
```

Comment:
I had to handle verify and cert arguments differently. Verify can be set to a default (True) and cert should not even be in the argument list if not set to a String(file path). I just wanted to motivate why they are handled differently.